notes/macOS.md

```diff
@@ -1,10 +1,16 @@
 ### -dynamic flag on macOS
```

Comment:
Let's mention -threaded here as well

Reply:
Oh, definitely.

What about [here](https://github.com/reflex-frp/reflex-platform/blob/f1b1d06511c95654c7e776c58bbbd638bb4dfc3e/try-reflex#L32) ?
Which flags should be recommended? Should it be os-specific?